backend/resources/hacknight.py

```diff
@@ -91,13 +91,13 @@ def delete(self, id):
         hacknight = Hacknight.query.get_or_404(id)
         participants = {participant.id for participant in hacknight.participants}
         json_data = request.get_json(force=True)
-        ids_schema = Schema.from_dict({"participants_ids": fields.List(fields.Int())})
+        ids_schema = Schema.from_dict({"participant_id": fields.List(fields.Int())})
```

Comment:
We should change participants_ids -> participant_id in post method too. Same for all unit tests in which we have "participants_ids". After those changes all tests should pass.